cypress/plugins/auth/authMSAL.config.js

```diff
@@ -15,9 +15,11 @@ if (!CY_MSAL_CLIENT_SECRET)
   console.warn('MSAL client secret is empty ! Please set CY_MSAL_CLIENT_SECRET env variable when running cypress');
 
 const CY_MSAL_API_HOSTNAME_FOR_SCOPE = process.env.CY_MSAL_API_HOSTNAME_FOR_SCOPE;
-if (!CY_MSAL_API_HOSTNAME_FOR_SCOPE)
+const CY_MSAL_SCOPE = process.env.CY_MSAL_SCOPE;
+if (!CY_MSAL_API_HOSTNAME_FOR_SCOPE && !CY_MSAL_SCOPE)
   console.warn(
-    'API hostname for MSAL scope is empty ! Please set CY_MSAL_API_HOSTNAME_FOR_SCOPE env variable when running cypress'
+    'API hostname for MSAL scope is empty ! Please set CY_MSAL_SCOPE or CY_MSAL_API_HOSTNAME_FOR_SCOPE env variable ' +
```

Comment:
_empty !_ is french, no spaces needed in English :slightly_smiling_face:

Reply:
Fixed all occurrences in this file :slightly_smiling_face: